app/views/admin/profiles/admin_profiles/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= render TabsComponent.new do |component| %>
```

Comment:
Soo, do we need these pages?

Reply:
No - I'm deleting them once I'll double-check everything - keep it as last step

Reply:
Deleted